client/my-sites/sidebar-unified/item.jsx

```diff
@@ -36,13 +39,24 @@ export const MySitesSidebarUnifiedItem = ( {
 	continueInCalypso,
 } ) => {
 	const reduxDispatch = useDispatch();
-	const isHappychatSessionActive = useSelector( ( state ) => hasActiveHappychatSession( state ) );
+	const isHappychatSessionActive = useSelector( hasActiveHappychatSession );
+	const siteId = useSelector( getSelectedSiteId );
+	const isJetpack = useSelector( ( state ) => isJetpackSite( state, siteId ) );
+	const isSiteAtomic = useSelector( ( state ) => isSiteWpcomAtomic( state, siteId ) );
+	const isJetpackNonAtomicSite = isJetpack && ! isSiteAtomic;
```

Comment:
Can we move this to `client/my-sites/sidebar-unified/index.jsx` and pass it here `isJetpackNonAtomicSite` as prop? This component renders many times and these changes are invariant of this component. Probably we can do the same with `isHappychatSessionActive`.

Reply:
Good idea! Done in https://github.com/Automattic/wp-calypso/pull/50444/commits/3c8c6d27ef3957a03beea37633a33e0fbeac7c09